platform/config/Env.ts

```diff
@@ -1,8 +1,11 @@
 import * as process from 'process';
 import { resolve } from 'path';
 
+import { Proxy } from '../server/http/Proxy/Proxy';
+
 interface WithConfig {
   config?: string;
+  kbnServer?: any;
```

Comment:
Don't really know where to plug this `kbnServer` in - `env` was the easiest one :)

Reply:
It works for now. I might want to move it later, but for the PoC it's :+1: